src/SonarScanner.MSBuild.Shim/JsonPropertiesWriter.cs

```diff
@@ -18,16 +18,271 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using Newtonsoft.Json;
```

Comment:
to review this file - it's better to check the diff against the PropertiesWriter.